.gitlab-ci/pre-commit-dynamic-stub.yml

```diff
@@ -0,0 +1,17 @@
+---
+# stub pipeline for dynamic generation
+pre-commit:
+  tags:
+  - light
+  image: 'ghcr.io/pre-commit-ci/runner-image@sha256:aaf2c7b38b22286f2d381c11673bec571c28f61dd086d11b43a1c9444a813cef'
```

Comment:
Is it more readable by changing the `ghcr.io/pre-commit-ci/runner-image@sha256:aaf2c7b38b22286f2d381c11673bec571c28f61dd086d11b43a1c9444a813cef`
to `ghcr.io/pre-commit-ci/runner-image:2024-05-11-4bb81b2-full`